ui/src/shared/utils/useVisDomainSettings.ts

```diff
@@ -79,7 +79,11 @@ export const useVisYDomainSettings = (
   timeRange: TimeRange | null = null
 ) => {
   const initialDomain = useMemo(() => {
-    if (storedDomain === null || storedDomain.every(val => val === null)) {
+    if (
+      storedDomain === undefined ||
+      storedDomain === null ||
+      storedDomain.every(val => val === null)
```

Comment:
suggestion: let the language do this work for you. we thought it could only be `null` but it ended up being `undefined` - i get the sense that it's not important what the value is - as long as it's not an array, or an array with no values, it should fall into this loop: 

```suggestion
      !Array.isArray(storedDomain) ||
      storedDomain.every(val => val === null)
```

Reply:
I hear what you're saying, but if that is in fact the case that we don't care about what the value is, shouldn't we just handle every falsey value?

`!storedDomain || storedDomain.every ... `

?